docs/_install-and-build/Backup--restore.md

```diff
@@ -68,4 +70,4 @@ geth import <filename>
 _See https://github.com/ethereum/wiki/wiki/Blockchain-import-export for more info_
 
 
-And finally: **DO NOT FORGET YOUR PASSWORD** and **BACKUP YOUR KEYSTORE**
\ No newline at end of file
+And finally: **DO NOT FORGET YOUR PASSWORD** and **BACKUP YOUR KEYSTORE**
```

Comment:
Positive language os often better than negative. I thinking it might also be good to set up some form of popout formatting for these notes and warnings.